app/controllers/application_controller.rb

```diff
@@ -120,6 +122,10 @@ def authenticate_user!
     end
   end
 
+  def set_default_view_variables
+    @adv_search = Query::Proposal::FieldedSearch.new({})
```

Comment:
I think we should continue to have search on "My Requests" and add it to "My Dashboard". For the New Request page and Request Details it seems like it would be a confusing context switch to move from a detail view to a search results page so let's not include search on those pages. @phirefly what do you think?